massa-async-pool/src/changes.rs

```diff
@@ -237,4 +254,13 @@ impl AsyncPoolChanges {
     pub fn push_delete(&mut self, msg_id: AsyncMessageId) {
         self.0.push(Change::Delete(msg_id));
     }
+
+    /// Pushes a message activation to the list of changes.
+    /// No add/delete compensations are done.
```

Comment:
Can we add more description about what is compensations here?

Reply:
It shouldn't be here. Old copy paste.